apps/emqx_bridge_rabbitmq/rebar.config

```diff
@@ -0,0 +1,33 @@
+%% -*- mode: erlang; -*-
+{erl_opts, [debug_info]}.
+{deps, [
+        %% The following two are dependencies of rabbit_common
```

Comment:
Just curiosity: I'd think that rebar3 would fetch dependencies of dependencies without the need to explicitly define them downstream. :thinking: 

Is this required due to `git_subdir` being used?

Reply:
It would if there is a rebar.conf file which does not exist in this case. Also even if there was a `rebar3.conf` file it would point to a hex package or an external git repository and we don't want that, right?